Microsoft.Teams.Apps.Scrum/Bots/ScrumBot.cs

```diff
@@ -477,7 +518,13 @@ private async Task CreateScrumAsync(string trailCardId, string scrumCardId, stri
         private async Task<Dictionary<string, string>> GetActivityIdOfMembersInScrum(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
         {
             var membersActivityIdMap = new Dictionary<string, string>();
-            var members = await TeamsInfo.GetMembersAsync(turnContext, cancellationToken);
+            bool isCacheEntryExists = this.memoryCache.TryGetValue(TeamMembersCacheKey, out List<TeamsChannelAccount> members);
+            if (!isCacheEntryExists)
+            {
+                members = await this.GetTeamMembersAsync(turnContext, cancellationToken);
+                this.memoryCache.Set(TeamMembersCacheKey, members, TimeSpan.FromDays(1));
+                this.telemetryClient.TrackTrace($"Total Members in group chat is :{members.Count()}");
+            }
 
             foreach (var member in members)
             {
```

Comment:
string mentionEntityText = string.Format("<at>{0}</at>", mentionedEntity.Name);

This needs to be HTML encoded.

Reply:
Encoded mention entity text.